include/vcpkg/commands.add-version.h

```diff
@@ -1,13 +1,9 @@
 #pragma once
 
-#include <vcpkg/commands.interface.h>
+#include <vcpkg/fwd/vcpkgcmdarguments.h>
+#include <vcpkg/fwd/vcpkgpaths.h>
 
 namespace vcpkg::Commands::AddVersion
```

Comment:
Are we leaving the whole command to get its own namespace vs just belonging to the `Commands` namespace thing alone until we discuss it?